src/MessageQueue/MessageQueue.php

```diff
@@ -109,6 +107,12 @@ private function doFlush(bool $async)
             foreach ($messagesToPush as $key => $message) {
                 $messageParameters = [
                     'Id'           => $key, // Identifier of the message in the batch
+                    'MessageAttributes' => [
+                        'name' => [
+                            'DataType'    => 'String',
+                            'StringValue' => $message['name'],
```

Comment:
haha, I think this is for casting the `StringValue` to the declared `DataType` when pulling messages from SQS?